examples/data-sources/prefect_account_members/data-source.tf

```diff
@@ -1,2 +1,9 @@
-# Query all Members in Account
+terraform {
+  required_providers {
+    prefect = {
+      source = "prefecthq/prefect"
+    }
+  }
+}
+
```

Comment:
Same here, the provider config is implied, and we don't specify this in any of the other examples.